src/researches/largestKeywordDistance.js

```diff
@@ -0,0 +1,70 @@
+const getIndicesByWord = require( "../stringProcessing/indices" ).getIndicesByWord;
+const forEach = require( "lodash/forEach" );
+const sortBy = require( "lodash/sortBy" );
+const normalizeQuotes = require( "../stringProcessing/quotes.js" ).normalize;
+
+/**
+ * Calculates the largest percentage of the text without a keyword.
+ *
+ * @param {Paper} paper The paper to check the keyword distance for.
+ * @returns {number} Returns the largest percentage of the text between two keyword occurrences
+ *                   or a keyword occurrence and the start/end of the text.
+ */
+module.exports = function( paper ) {
```

Comment:
Is there a specific reason that this is a directly exported function vs a class or something else?